Library/Homebrew/cmd/reinstall.rb

```diff
@@ -58,22 +61,22 @@ def reinstall
 
     FormulaInstaller.prevent_build_flags unless DevelopmentTools.installed?
 
-    Install.perform_preinstall_checks
+    perform_preinstall_checks
```

Comment:
I like knowing which module this came from

Reply:
See https://github.com/Homebrew/brew/pull/8084#discussion_r460737938.